lib/squake/products.rb

```diff
@@ -2,6 +2,7 @@
 # frozen_string_literal: true
 
 # https://docs-v2.squake.earth/operation/operation-get-products
+
```

Comment:
NIT: should we move the comment above the `class` (not the module) so the IDE can show this when hovering over the class?